compiler/noirc_frontend/src/tests/aliases.rs

```diff
@@ -75,3 +75,74 @@ fn double_generic_alias_in_path() {
     "#;
     assert_no_errors(src);
 }
+
+#[test]
+fn identity_numeric_type_alias_works() {
+    let src = r#"
+    pub type Identity<let N: u32>: u32 = N;
+    "#;
+    assert_no_errors(src);
+}
```

Comment:
I've moved the tests here into the `aliases` module to scope them a bit more.

I've also added this test which checks that we can have a type alias which just returns its generic parameter which seems to fail.